src/main/webapp/app/stores/firebase/firebase-data.store.ts

```diff
@@ -3,7 +3,7 @@ import { action, makeObservable, observable } from 'mobx';
 import FirebaseAppStore from './firebase-app.store';
 
 export class FirebaseDataStore<T> {
-  public data: Readonly<T> = undefined;
+  public data: Readonly<T> | undefined = undefined;
```

Comment:
Does TS treat undefined and null the same? `onValue` will return `null` if there is no data located at the provided path. Maybe using null is more accurate in this case.

Reply:
they are not the same.
I'll change it to null